guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

```diff
@@ -1,18 +1,22 @@
 [id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
 = Changing the Repository Sets Status in {Project}
```

Comment:
Since this procedure applies to hosts (and not AKs or bulk hosts), should this info be added to the title?
```suggestion
= Changing the Repository Sets Status for a Host in {Project}
```